### PR TITLE
ADD METHOD isRewardedVideoAvailable

### PR DESCRIPTION
**Pull request by Insomniiak**
Adding the method `isRewardedVideoAvailable()`

IS #42

### CODE REVIEW

File: RNIronSourceRewardedVideo.js [hunk 1]
```diff
@@ -60,10 +60,18 @@ const removeAllListeners = () => {
...
+  setTimeout(() => Promise.reject(new Error(`Timeout isRewardedVideoAvailable: timeout: ${timeoutTime}`))
+    , timeoutTime);
+  return RNIronSourceRewardedVideo.isRewardedVideoAvailable();
+}
+
 module.exports = {
   ...RNIronSourceRewardedVideo,
   initializeRewardedVideo: () => RNIronSourceRewardedVideo.initializeRewardedVideo(),
   showRewardedVideo: (placementName) => RNIronSourceRewardedVideo.showRewardedVideo(placementName),
+  isRewardedVideoAvailable: () => RNIronSourceRewardedVideo.isRewardedVideoAvailable(callback),
```
Comment (StanislavMayorov) [inline RNIronSourceRewardedVideo.js]: What is `callback`?

File: RNIronSourceRewardedVideo.js [hunk 2]
```diff
@@ -60,10 +60,18 @@ const removeAllListeners = () => {
   IronSourceRewardedVideoEventEmitter.removeAllListeners('ironSourceRewardedVideoAdRewarded');
 };
 
+isRewardedVideoAvailable = async () => {
+  const timeoutTime = 5000;
+  setTimeout(() => Promise.reject(new Error(`Timeout isRewardedVideoAvailable: timeout: ${timeoutTime}`))
+    , timeoutTime);
+  return RNIronSourceRewardedVideo.isRewardedVideoAvailable();
+}
```
Comment (StanislavMayorov) [inline RNIronSourceRewardedVideo.js]: Why do you need timer?

Comment (StanislavMayorov) [reply RNIronSourceRewardedVideo.js]: Where is this function used?

File: .gitignore [hunk 3]
```diff
@@ -3,6 +3,9 @@
 *.moved-aside
 *.xcuserstate
 *.xcscmblueprint
+.project
+.gradle
+.settings
```
Comment (StanislavMayorov) [inline .gitignore]: When are these files created? Is it for `ExampleApp`?

**Comment by StanislavMayorov** [comment]
Hi! Thank you for contribution.

**Comment by StanislavMayorov** [comment]
Readme have to be updated too.

File: RNIronSourceRewardedVideo.js (hunk 1 above)
Comment (Insomniiak) [reply RNIronSourceRewardedVideo.js]: Oh myBad I didn't pushed my last modification.

**Comment by Insomniiak** [comment]
And One more Thing, I don't have IOS on my so I can't try right now if its really working on IOS.

**Comment by Insomniiak** [comment]
@StanislavMayorov If you got any questions

**Comment by StanislavMayorov** [comment]
I have updated your branch to fix requested changes

**Pull request merged**